_pages/how-we-work/leading-projects.md

```diff
@@ -25,6 +26,14 @@ In all this, your closest collaborators are 1) your account manager and 2) your
 
 Someone (the partner, 18F leadership, or others) might ask you about how the project is going, and 18F will expect you to have an answer. It does not mean you should _do_ all of these things yourself — the most senior project leads are often those who can speak confidently about how _others_ are owning various aspects of the work.
 
+### Rotating tasks among teammates
+
+The project lead is not the only one responsible for administrative tasks such as scheduling meetings with partners and taking notes. We encourage project leads to set up systems for rotating this work. For example, a team might have a schedule that indicates who is responsible for sending a progress update email to the partner each week.
```

Comment:
I think we can make a stronger (and shorter) argument here:

> Teams are more resilient and equitable when administrative tasks like scheduling meetings or taking notes are shared. It can help to proactively set up systems to rotate this work (for example: a schedule for who will send weekly ship emails each week).

Reply:
👌